### ISSUE TITLE
Compile-AppInNavContainer - Timeout when downloading symbols

### ISSUE THREAD

**Issue by babaczyk** [microsoft/navcontainerhelper#1077]
Randomly I have a timeout issue in my CI pipelines. Timeouts exceeded when Compile-AppInNAVConainer tries to download symbols using WebClient. Therefore, I have a question - is it possible to set a timeout value for this script?

```
2020-07-06T09:01:46.8846143Z ##[section]Starting: Compile App
2020-07-06T09:01:47.0576521Z ==============================================================================
2020-07-06T09:01:47.0577097Z Task         : PowerShell
2020-07-06T09:01:47.0577589Z Description  : Run a PowerShell script on Linux, macOS, or Windows
2020-07-06T09:01:47.0578107Z Version      : 2.170.1
2020-07-06T09:01:47.0578495Z Author       : Microsoft Corporation
2020-07-06T09:01:47.0579018Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/powershell
2020-07-06T09:01:47.0579608Z ==============================================================================
2020-07-06T09:01:51.9763945Z Generating script.
2020-07-06T09:01:51.9849576Z Formatted command: . 'C:\azure-vstsagent\Build1\_work\1\s\scripts\Compile-App.ps1' -credential ([PSCredential]::new("BCSRV$", (ConvertTo-SecureString -String "***" -AsPlainText -Force))) -buildProjectFolder "C:\azure-vstsagent\Build1\_work\1\s" -buildArtifactFolder "C:\azure-vstsagent\Build1\_work\1\a" -updateSymbols:$false -appVersion "2.0.58.0"
2020-07-06T09:01:52.0516122Z ========================== Starting Command Output ===========================
2020-07-06T09:01:52.0893015Z ##[command]"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command ". 'C:\azure-vstsagent\Build1\_work\_temp\bf547af3-6510-4d58-8f46-ab39a775d426.ps1'"
2020-07-06T09:02:01.8370893Z Using Version 2.0.58.0
2020-07-06T09:02:02.1593073Z Compiling xxxx
2020-07-06T09:02:05.6931857Z Using Symbols Folder: C:\azure-vstsagent\Build1\_work\1\s\.alPackages
2020-07-06T09:03:15.6837417Z Downloading symbols: Microsoft_Application_14.1.32615.0.app
2020-07-06T09:03:15.8453433Z Url : http://172.29.28.249:7049/NAV/dev/packages?publisher=Microsoft&appName=Application&versionText=14.0.0.0&tenant=default
2020-07-06T09:03:33.7792153Z Downloading symbols: Microsoft_System_14.0.32600.0.app
2020-07-06T09:03:33.7816769Z Url : http://172.29.28.249:7049/NAV/dev/packages?publisher=Microsoft&appName=System&versionText=14.0.0.0&tenant=default

**2020-07-06T09:05:14.7433038Z ##[error]Compile-AppInBCContainer : Exception calling "DownloadFile" with "2" argument(s): "UpÅ‚ynÄ…Å‚ limit czasu operacji"** /// It's mean Timeout in English

At C:\azure-vstsagent\Build1\_work\1\s\scripts\Compile-App.ps1:45 char:16

+ ...  $appFile = Compile-AppInBCContainer -containerName $containerName -c ...

+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

    + CategoryInfo          : NotSpecified: (:) [Compile-AppInNavContainer], MethodInvocationException

    + FullyQualifiedErrorId : WebException,Compile-AppInNavContainer
2020-07-06T09:05:14.8174309Z ##[error]PowerShell exited with code '1'.
2020-07-06T09:05:14.8189558Z ##[section]Finishing: Compile App
```

**Comment by freddydk** [comment]
09:05:14 - 09:03:33 is 106 seconds and default timeout on webclient download is 100 seconds - so yes, it looks like you are hitting that. I will increase the timeout.

**Comment by babaczyk** [comment]
Thanks a lot!

**Comment by freddydk** [comment]
Shipped in 0.7.0.12
